warp/payload/payload.go

```diff
@@ -6,22 +6,22 @@ package payload
 import (
 	"fmt"
 
-	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ethereum/go-ethereum/common"
 )
 
 // AddressedPayload defines the format for delivering a point to point message across VMs
 // ie. (ChainA, AddressA) -> (ChainB, AddressB)
 type AddressedPayload struct {
-	SourceAddress      ids.ID `serialize:"true"`
-	DestinationChainID ids.ID `serialize:"true"`
-	DestinationAddress ids.ID `serialize:"true"`
-	Payload            []byte `serialize:"true"`
+	SourceAddress      common.Address `serialize:"true"`
+	DestinationChainID common.Hash    `serialize:"true"`
```

Comment:
why do we change this from `ids.ID` to common.Hash?

Reply:
It's a no-op change from one alias to the other, but changed it to `common.Hash` to stick with using the `common` package instead of one from each.